src/EditorFeatures/TestUtilities/ServicesTestUtilities.csproj

```diff
@@ -94,6 +94,7 @@
     <InternalsVisibleToTest Include="Roslyn.Compilers.VisualBasic.Test.Utilities" />
     <InternalsVisibleToTest Include="Roslyn.InteractiveHost.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.UnitTests" />
+    <InternalsVisibleToTest Include="Roslyn.VisualStudio.Test.Utilities" />
```

Comment:
Yes, however, I imagine Roslyn specific components (such as InteractiveWindow) which require IVT should stay in a separate library in the Roslyn repo. The code we move out of the repo should be global (and likely only depend on the DTE and VSSDK).